src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js

```diff
@@ -25,10 +24,14 @@
             // Subscribe to the location update event
             $scope.$on('nih.navigation.locationUpdated', function(event, data) {
                 MapControl.trackUser(data);
-                angular.extend(ctl.map.center, {
-                    zoom: 19,
-                    lat: data[1],
-                    lng: data[0]
+                angular.extend(ctl.map, {
+                    bounds: {
```

Comment:
Weird, what does this fix?

These two properties interact really strangely in the leaflet directive sometimes.